kubernetes/services/middleman.yml.j2

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind:       Deployment
+metadata:
+  name: middleman
+spec:
+  replicas:             1
+  revisionHistoryLimit: 1
+  template:
+    metadata:
+      labels:
+        run: middleman
+    spec:
+      containers:
+        - name:             concent-api
+          image:            {{ docker_registry }}/{{image_prefix }}concent-api:{{ image_version }}
+          imagePullPolicy:  Always
+          command:          ["/usr/local/bin/concent-api-manage.sh", "middleman", "--bind-address", "0.0.0.0", "--internal-port", "9054"]#FIXME AFTER external-port was added, "--external-port 9055"]
+          ports:
+            - containerPort: 9054
+              name: internal-port
+            - containerPort: 9055
+              name: external-port
+          resources:
+            requests:
+              memory: "0.3Gi"
+              cpu:    "50m"
```

Comment:
Can we give it less than 300 MB? It's a really small app, I think it should require much less.

Reply:
I don't know. We can only estimate this value because the application is not finished yet.

Reply:
So maybe let's start with a lower value and we'll increase it if it's too low.